pkg/policies/binary/binary.go

```diff
@@ -172,23 +173,43 @@ func (b Binary) Check(ctx context.Context, c *github.Client, owner,
 		return nil, res.Error2
 	}
 
+	logs := convertLogs(l.logs)
+	pass := res.Score >= checker.MaxResultScore
 	var notify string
-	if res.Score < checker.MaxResultScore {
-		notify = fmt.Sprintf("Scorecard Check Binary Artifacts failed: %v\n"+
-			"Please run scorecard directly for details: https://github.com/ossf/scorecard\n",
+	if !pass {
+		notify = fmt.Sprintf(
+			"Scorecard Check Binary Artifacts failed: %v\n"+
+				"Binary Artifacts are an increased security risk in your repository. Binary artifacts cannot be reviewed, allowing possible obsolete or maliciously subverted executables.\n"+
+				"To remediate, remove the generated executable artifacts from the repository. Build from source where possible.\n"+
+				"For more information see https://github.com/ossf/scorecard/blob/main/docs/checks.md#binary-artifacts. Also, you may run scorecard directly on this repository for more details.\n",
```

Comment:
replace "scorecard" with "Security Scorecards" and link if possible?

Reply:
Fixed